shared/xor_seed.py

```diff
@@ -66,32 +69,37 @@ async def xor_split_start(*a):
             if sv.mode == 'words':
                 words = bip39.b2a_words(sv.raw).split(' ')
 
-            if not words or len(words) != 24:
-                await ux_show_story("Need 24-seed words for this feature.")
+            if not words:
+                await ux_show_story("Need seed words for this feature.")
                 return
 
-            # checksum of target result is useful.
-            chk_word = words[-1]
+            # checksum of target result is useful (only for 24 words).
+            chk_word = None
+            if len(words) == 24:
+                chk_word = words[-1]
```

Comment:
this seems useful at any word length

Reply:
you're right - I somehow dismissed the idea in the beginning of the task thinking that we do not have enough bits to reconstruct. But the idea is wrong as we have the seed that we're trying to reconstruct at hand. Thanks - fixed